html/patterns/04-pages-01-one-sidebar-one-sidebar/04-pages-01-one-sidebar-one-sidebar.html

```diff
@@ -17,20 +17,37 @@
   </head>
   <body class="">
   
-<div class="sidebar">
-    <div class="section">
-    <div class="section-heading">
-    Categories
-</div>    <div class="section-content">
-    <ul><li><a href="">Tech</a></li><li><a href="">News</a></li><li><a href="">Tutorials</a></li><li><a href="">Sports</a></li><ul>
-</div></div>    <div class="section">
-    <div class="section-heading">
-    Sign Up
-</div>    <div class="section-content">
-    <p>Subscribe to our RSS feed today and be the first to get new updates</p>
-</div></div></div>
-<div class="content">
-    <div class="blog-list">
+<div class="page-wrapper">
```

Comment:
I wonder if this will break the Drupal implementation by adding a new auxiliary surrounding div tag. Happy to do this, if there is merit.